tests/renderering/text/Text.tests.ts

```diff
@@ -140,12 +140,11 @@ describe('Text', () =>
 
             const text = new Text({ text: 'foo' });
 
-            const key = text._getKey();
-
             container.addChild(text);
-
             renderer.render({ container });
 
+            const key = text._getKey();
```

Comment:
not much of a muchness - but no changes required to this file?

Reply:
the change is required as we now set the resolution in the pipe, so getting the key before its rendered would still give `null` and after it would be `1`